routes/baseActionsRouter.js

```diff
@@ -797,6 +797,59 @@ function parseTwoOptionVote(tx) {
 	}
 }
 
+/**
+ * Guess if transaction is a flipstarter transaction.
+ *
+ * Flipstarter transactions has all inputs signe as ALL|FORKID|ANYONECANPAY
+ * and is around ~4 sat/bytes in fee.
+ */
+function isFlipstarter(tx, fee) {
+  try {
+    if (fee < 3.9 || fee > 4.1) {
+      return false;
+    }
+    if (tx.vin[0].coinbase) {
+      return false;
+    }
+
+    for(let i = 0; i < tx.vin.length; i++) {
```

Comment:
maybe redundant, but can't we also add a check for an heuristic related to the minimum number of inputs for a successful. 

I mean it could be theoretically possible that one single pledge suffice to fund a single flipstarter, but at the best of my knowledge that has never happened.